libs/@hashintel/petrinaut/package.json

```diff
@@ -10,14 +10,17 @@
   "license": "(MIT OR Apache-2.0)",
   "sideEffects": false,
   "type": "module",
+  "main": "src/main.ts",
+  "types": "src/main.ts",
   "scripts": {
-    "build": "vite build",
-    "dev": "vite",
+    "build": "rimraf dist && tsc -p tsconfig.build.json",
```

Comment:
I guess in the future it would be better to build using Vite?

Reply:
We talked about this – `vite` (or some other bundling step) is needed to do the PandaCSS compilation. I'll add it back in